drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-api/src/main/java/org/drools/workbench/screens/guided/dtable/service/GuidedDecisionTableGraphEditorService.java

```diff
@@ -39,8 +39,7 @@
         SupportsCopy,
         SupportsRename {
```

Comment:
Should this not also now extend ```SupportsSaveAndRename```?

Reply:
Oh.. I see a new interface further down....

Reply:
;-)